LoopKit/InsulinKit/InsulinType.swift

```diff
@@ -64,15 +69,26 @@ public enum InsulinType: Int, Codable, CaseIterable {
             return LocalizedString("Lyumjev is a mealtime insulin lispro formulation with the addition of citrate and treprostinil made by Eli Lilly", comment: "Description for lyumjev insulin type")
         case .afrezza:
             return LocalizedString("Afrezza is an ultra rapid-acting mealtime insulin that is breathed in through your lungs using an oral inhaler and made by MannKind", comment: "Description for afrezza insulin type")
+        case .lantus:
+            return LocalizedString("Lantus is a long acting basal insulin that is injected either once or twice a day", comment: "Description for lantus insulin type")
         }
     }
     
     public var pumpAdministerable: Bool {
         switch self {
-        case .afrezza:
+        case .afrezza, .lantus:
             return false
         default:
             return true
         }
     }
+    
+    public var isBasal: Bool {
```

Comment:
Where is this being used? I assume you would *not* want these doses to affect Loop's calculations, right?  Since Loop has a relative view of basal.